Parser/WEB3/buyGTON.ts

```diff
@@ -1,29 +1,32 @@
 import Big from 'big.js';
 import Web3 from 'web3';
 import axios from 'axios';
-import { gtonAddress, usdcAddress } from '../../config/config';
+import { gtonAddress } from '../../config/config';
 import { fromWei } from '../WEB3/API/balance';
 
 import { validate } from './validate';
 
-// USDC -> GTON
+// swap on GTON
 
 declare const window: any;
 
-export const checkSwapAmount = async (amount: Big): Promise<string> => {
+export const checkSwapAmount = async (
+  amount: Big,
+  fromTokenAddress: string,
+  decimals: number,
+): Promise<string> => {
   await validate();
   try {
     let status = await axios.get('https://api.1inch.io/v4.0/1/healthcheck');
     if (status) {
       let quote = await axios.get(
-        `https://api.1inch.io/v4.0/1/quote?fromTokenAddress=${usdcAddress}&toTokenAddress=${gtonAddress}&amount=${amount}`,
+        `https://api.1inch.io/v4.0/1/quote?fromTokenAddress=${fromTokenAddress}&toTokenAddress=${gtonAddress}&amount=${amount}`,
       );
       return `
-      You will swap ${fromWei(quote.data.fromTokenAmount, 6)} $USDC on ${fromWei(
-        quote.data.toTokenAmount,
-      )} $GTON
+      You will swap ${fromWei(quote.data.fromTokenAmount, decimals)} $${
+        quote.data.fromToken.symbol
+      } on ${fromWei(quote.data.toTokenAmount)} $GTON
```

Comment:
for ${fromWei(quote.data.toTokenAmount)} $GTON

Reply:
Fixed